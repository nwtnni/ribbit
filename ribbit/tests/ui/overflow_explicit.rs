#[ribbit::pack(size = 32)]
struct Foo {
    #[ribbit(offset = 1)]
    a: u32,
}

fn main() {}
