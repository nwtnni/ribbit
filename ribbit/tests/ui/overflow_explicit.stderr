error: Field requires 32 bits at offset 1, but only 31 are available
 --> tests/ui/overflow_explicit.rs:3:23
  |
3 |     #[ribbit(offset = 1)]
  |                       ^
